financial-services-accelerator/accelerators/fs-is/repository/resources/wso2is-7.0.0-deployment.toml

```diff
@@ -284,6 +282,7 @@ additional_attribute_filter= "org.wso2.financial.services.accelerator.identity.e
 
 [oauth.oidc]
 id_token.signature_algorithm="PS256"
+enable_claims_separation_for_access_tokens = false
```

Comment:
is this a new config?why we need it..?

Reply:
without this config the custom jwt claims added are not getting added from the IS side in IS 7x. Need to clarify this behaviour from the IS team